v3/deployment.go

```diff
@@ -330,6 +331,58 @@ var _ = V3Describe("deployment", func() {
 				}).ShouldNot(ContainSubstring("Hi, I'm Dora"))
 			})
 		})
+
+		Context("instance-steps", func() {
+			BeforeEach(func() {
+				Expect(cf.Cf("scale", appName, "-i", "4").Wait(Config.CfPushTimeoutDuration())).To(Exit(0))
+			})
+
+			It("deploys an app, transitions to pause and can be continued multiple times and then deploys successfully", func() {
+				By("Pushing a canary deployment")
+				Expect(cf.Cf("push", appName, "--strategy", "canary", "--instance-steps=25,50,75", "--no-wait", "-b", Config.GetStaticFileBuildpackName(), "-p", assets.NewAssets().Staticfile).Wait(Config.CfPushTimeoutDuration())).To(Exit(0))
+
+				for i := 1; i <= 3; i++ {
```

Comment:
Do you want to include any validation that the step percentage is working? I notice that in the test above for instance, the `cf app` command output is used to gauge how many instances are running vs. starting. Is there some similar output that could be used here?

Reply:
Good idea, that output was not actually present when this test was written but has since been added. I'll update.